products/proxmox-ve.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Proxmox VE
+category: os
+iconSlug: proxmox
+permalink: /proxmox-ve
+alternate_urls:
+-   /pve
+-   /proxmoxve
+activeSupportColumn: Supported
+releasePolicyLink: https://pve.proxmox.com/wiki/Roadmap
+releaseDateColumn: true
+eolColumn: true
+discontinuedColumn: false
+versionCommand: pveversion –v
+
+releases:
+-   releaseCycle: "7.x"
+    support: true
+    eol: false
+    latest: "7.3"
+    link: https://pve.proxmox.com/wiki/Roadmap#Proxmox_VE_7.3
```

Comment:
A `changelogTemplate` would be better. With a `changelogTemplate` you don't have to set the link for each release cycle.

In this case the following `changelogTemplate` should work :

```
changelogTemplate: "https://pve.proxmox.com/wiki/Roadmap#Proxmox_VE___RELEASE_CYCLE__"
```

See also: https://github.com/endoflife-date/endoflife.date/blob/master/CONTRIBUTING.md#new-adding-a-new-product.